### ISSUE TITLE
In regtest mode with bitcoind backend, wallet does not pick up coinbase outputs

### ISSUE THREAD

**Issue by GambolingPangolin** [sparrowwallet/sparrow#265]
## Setup

* Start a regtest node
* Start sparrow with `-n regtest` and configure it to connect to the node we just started
* Create a P2WPKH wallet with private keys
* Run `bitcoin-cli -regtest generatetoaddress 110 $sparrow_address` 

## Expected behavior

110 new UTXOs should appear in the wallet, with 10 spendable and 100 in a mined, but unspendable state.

## Actual behavior

No transactions show up in the wallet.  This may also occur on mainnet.  I don't have a way to check.

## System information

* Sparrow 1.5.1
* Bitcoin Core 22.0
* Ubuntu 20.04

**Comment by craigraw** [comment]
> The Electrum RPC protocol doesn't have a way to specify whether an output is mature or not

Agreed - I think that the maturity of the output should be ignored and simply listed as unspent - which is correct. The wallet software can handle the coin selection criteria.

> Would Sparrow handle this correctly?

Indeed it did not. I've now implemented this in https://github.com/sparrowwallet/drongo/commit/57290a20a143bc0f0d8440fb3c14592c2e204e6b and 237f978.

**Comment by kdmukai** [comment]
Bump! Behavior seems to be the same in Sparrow v1.8.1. Neither mature nor immature coinbase outputs are found.

Same setup: bitcoind (v24.0.1) in regtest.

Is there a workaround?

**Comment by craigraw** [comment]
On the list to add to Cormorant (Sparrow's Bitcoin Core RPC library) for the next version. The workaround for now is to use an Electrum server.

**Comment by craigraw** [comment]
Closing this old issue, coinbase outputs are supported in Cormorant.
